.azure/pipelines/ci.yml

```diff
@@ -134,15 +126,6 @@ stages:
       # This is intentional to workaround https://github.com/dotnet/arcade/issues/1957 which always re-submits for code-signing, even
       # if they have already been signed. This results in slower builds due to re-submitting the same .nupkg many times for signing.
       # The sign settings have been configured to
-      - ${{ if ne(variables['System.TeamProject'], 'public') }}:
```

Comment:
Why remove this? We'll have to add it back in if/when we do internal builds of net5 (CC @mmitche)

Reply:
Yes, this should remain or it will have to be added back later.

Reply:
I didn't remove it; I just moved the same steps up in default-build.yml to make sure it happens before running `./restore.cmd`. The change also removes some duplication in the YAML.

Reply:
@wtgodbe @mmitche put another way, this is no longer necessary anywhere the default-build.yml template is used.

Is this conversation resolved?